docs/rst/getting_started/building.rst

```diff
@@ -283,44 +322,11 @@ the embedded Mitsuba version number to the terminal:
 
 .. _sec-getting_started-building-setup_automation:
 
-Setup automation
-----------------
-
-Conda environment creation can be automatically handled by executing the
-``resources/envs/conda_create_env.sh`` script at the root of the Eradiate source
-tree.
-
-.. warning::
-
-   Executing the Conda environment setup script will reset the existing
-   environment!
-
-The script automates by default the default Conda environment setup and the
-Eradiate package installation. Optional steps can also be automate using a
-series of flags:
-
--d    Perform :ref:`development dependency installation <sec-getting_started-building-python-conda-optional>`
--j    Perform :ref:`Jupyter lab installation and extension activation <sec-getting_started-building-python-conda-optional>`
--a    Add automatic environment variable setup to environment activation script
-      (makes the :ref:`environment variable setup <sec-getting_started-building-environment_variables>`
-      no longer necessary)
--e    Add a `direnv <https://direnv.net/>`_  ``.envrc`` file to the root of the
-      Eradiate source tree (makes the
-      :ref:`environment variable setup <sec-getting_started-building-environment_variables>`
-      no longer necessary)
-
-.. tabbed:: Typical user setup
+Required data sets
+------------------
 
-   .. code-block:: bash
-
-      bash resources/envs/conda_create_env.sh -j -a
-
-.. tabbed:: Typical developer setup
-
-   .. code-block:: bash
-
-      bash resources/envs/conda_create_env.sh -d -j -a
-
-.. note::
-
-   We are not sourcing the script, we are executing it in a subshell.
+Eradiate does not automatically ship all available data sets due to their size.
+In order to successfully run all tests and tutorials, at least the ``us76_u86_4-fullrange``
```

Comment:
The ``us76_u86_4-fullrange`` data set is not required to run tests, right?
I'm curious, what tutorials require this dataset? I think the OneDimSolver tutorial requires it but that's the only one, no?

Reply:
We talked about that already so I'll just close this

Reply:
If the xarray tutorial in the user guide requires that the `us76_u86_4` datasets are downloaded, maybe we should add to that tutorial a reference to the section `_sec-user_guide-manual_download` in the data guide?